packages/expo-av/android/build.gradle

```
@@ -101,3 +166,36 @@ dependencies {
 
   testImplementation 'io.mockk:mockk:1.9'
 }
+
+task extractAARHeaders {
+  doLast {
+    configurations.extractHeaders.files.each {
+      def file = it.absoluteFile
+      copy {
+        from zipTree(file)
+        into "$buildDir/$file.name"
+        include "**/*.h"
+      }
+    }
+  }
+}
+
+task extractJNIFiles {
+  doLast {
+    configurations.extractJNI.files.each {
+      def file = it.absoluteFile
+      copy {
+        from zipTree(file)
+        into "$buildDir/$file.name"
+        include "jni/**/*"
+      }
+    }
+  }
+}
+
+tasks.whenTaskAdded { task ->
+  if (task.name.contains('externalNativeBuild')) {
```

```suggestion
  if (task.name.contains('externalNativeBuild') || task.name.startsWith('configureCMake')) {
```
we are going to upgrade react-native 0.68 soon. it has AGP 7 and the cmake task setup is getting a little different.